src/clients/app-client.js

```diff
@@ -1,12 +1,15 @@
 'use strict'
 
-const Client = require('./base-browser-client')
+const Client = require('./base-client')
 
 class AppClient {
-  constructor (url) {
+  constructor (url, options = {}) {
     this.client = new Client({
       name: 'AppsClient',
       url: url,
+      accessKey: options.accessKey,
+      secretKey: options.secretKey,
+      doorkeeperBaseUrl: options.doorkeeperBaseUrl,
       contentType: 'application/vnd.api+json'
     })
```

Comment:
I would put `accessKey`, `secretKey`, `doorkeeperBaseUrl` (as `baseUrl`) and `userId` into a `doorkeper` object.

Reply:
Good idea, but I dislike the strong connection to `doorkeeper`, because I don't know how long we will keep our token handling there.

Instead of `doorkeeper` I would suggest `serviceToken`.

```
new Client(url, {
  contentType: '...',
  serviceToken: {
    accessKey: 'foo123',
    userId: 'subscription_system',
    ...
  }
})
```